src/persist-client/src/internal/apply.rs

```diff
@@ -264,10 +273,15 @@ where
             cmd.seconds.inc_by(now.elapsed().as_secs_f64());
 
             match ret {
-                ApplyCmdResult::Committed((seqno, new_state, res, maintenance)) => {
+                ApplyCmdResult::Committed((seqno, diff, new_state, res, maintenance)) => {
                     cmd.succeeded.inc();
                     self.shard_metrics.cmd_succeeded.inc();
                     self.update_state(new_state);
+                    if let Some(pubsub_sender) = self.pubsub_sender.as_ref() {
+                        if self.cfg.dynamic.pubsub_push_enabled() {
```

Comment:
I think we probably also want this cfg to short circuit the broadcast in the service running in envd